scrapers/getWorldometers.js

```diff
@@ -3,13 +3,14 @@ const cheerio = require('cheerio');
 const countryUtils = require('../utils/country_utils');
 const logger = require('../utils/logger');
 
-/**
- * Gets country data from table cell on worldometers website
- * @param 	{Object} 	cell 	Country table cell from worldometers website
- * @returns {string} 			Country name
- */
-const getCountryData = (cell) => (cell.children[0].data || cell.children[0].children[0].data || cell.children[0].children[0].children[0].data
-	|| cell.children[0].children[0].children[0].children[0].data || (cell.children[0].next.children[0] && cell.children[0].next.children[0].data) || '').trim();
+const columnMapper = ['cases', 'todayCases', 'deaths', 'todayDeaths', 'recovered', 'active', 'critical', 'casesPerOneMillion', 'deathsPerOneMillion', 'tests', 'testsPerOneMillion'];
```

Comment:
just call this `columns`, `columnMapper` sounds like a function name

Reply:
exactly